### PR TITLE
Stack DS

### PR DESCRIPTION
**Pull request by num-js**

### CODE REVIEW

**Comment by num-js** [comment]
image

File: Stack/stack.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <h3>Stack-DS</h3>
+        <p>(Open console to check output)</p>
+
+        <br /><br />
+
+        <input id="value" type="text" />
+        <button onclick="pushData()">Push</button>
+
+        <br /><br />
+        <button onclick="pop()">pop</button>
```
Comment (numwebdev) [inline Stack/stack.html]: make 'p' capital in pop
like - Pop

Comment (num-js) [reply Stack/stack.html]: Done

**Review by numwebdev — APPROVED** [review]
Looks Good
Ready to Merge.

**Pull request merged**